### DISCUSSION TITLE
Line break in donut label not working after hovering the chart in apexchart

### DISCUSSION THREAD

**Discussion by wahibshaikh** [apexcharts/apexcharts.js#3655] [Q&A]
I want to add line break in the label of donut chart. It shows line break on initial render, but after hovering the chart it separates the label with comma(,) . I'm adding the configs and screenshots.

Initial render:
![initial](https://user-images.githubusercontent.com/93134131/218658021-e2bd1a37-fd8d-4102-92b9-90ef71558b01.png)

After Hover:
![hover](https://user-images.githubusercontent.com/93134131/218658255-a6b9711c-1182-48e7-9ce4-7332f53b745f.png)

Config: 
```
{
    series: [4, 5, 6, 7],
    options: {
      plotOptions: {
        pie: {
          donut: {
            labels: {
              show: true,
              name: {
                show: true,
                color: '#000000',
                fontWeight: 600,
                formatter: (val) => {
                  return val
                },
              },
              value: {
                show: true,
              },
              total: {
                show: Boolean(questionCount),
                formatter: () => questionCount || 0,
              },
            },
          },
        },
      },
      chart: {
        type: type || 'donut',
      },
      dataLabels: {
        enabled: false,
      },
      legend: {
        show: false,
      },
      tooltip: {
        marker: { show: true },
        fillSeriesColor: false,
        theme: 'light',
      },

      colors: colors || ['#008000', '#ffa500', '#ff0000', '#000000'],
      responsive: [
        {
          breakpoint: 480,
          options: {
            chart: {
              width: 300,
              height: 300,
            },
            legend: {
              display: false,
              position: 'bottom',
            },
          },
        },
      ],
      labels: labels || [
        'Correct',
        ['Partially', 'Correct'],
        'Incorrect',
        'Unattempted',
      ],
    },
  }
```

**Comment by mohsin-polymer** [comment]
I am getting same problem. Even disabling hover effect doesn't solve the problem.
Here is how I have disabled hover effect:

`states: {
                hover: {
                    enabled: false,
                    filter: {
                        type: 'none',
                    }
                },
            },`

**Comment by tmcdos** [comment]
I have the same problem. Is there any solution ?

**Reply by ericlara22** [reply]
The only thing I can think of is to disable the pointer events on the parent component
